dojo/settings/settings.dist.py

```diff
@@ -1563,46 +1563,46 @@ def saml2_attrib_map_format(dict):
     'loggers': {
         'django.request': {
             'handlers': ['mail_admins', 'console'],
-            'level': '%s' % LOG_LEVEL,
+            'level': f'{LOG_LEVEL}',
```

Comment:
Might not need the `str()` but definitely don't need a format string

```suggestion
            'level': str(LOG_LEVEL),
```